src/types/num.rs

```diff
@@ -261,9 +270,14 @@ mod slow_128bit_int_conversion {
                             ffi::PyNumber_Index(ob.as_ptr()),
                         )
                     }?;
-                    let mut bytes = [0u8; std::mem::size_of::<$rust_type>()];
+                    let mut bytes = [0u8; BYTE_SIZE];
+                    let kwargs = if $is_signed {
+                        Some([("signed", true)].into_py_dict(py))
+                    } else {
+                        None
+                    };
                     let pybytes: &PyBytes = num
-                        .call_method("to_bytes", (bytes.len(), "little"), kwargs(py, $is_signed))?
+                        .call_method("to_bytes", (bytes.len(), "little"), kwargs)?
```

Comment:
this seems to revert a refactoring, or did you want to change the FromPy part as well?

Reply:
Since now the `kwargs` function is called once, I moved it in the function.